### DISCUSSION TITLE
Exist Beta builder version?

### DISCUSSION THREAD

**Discussion by gabfus** [albar965/littlenavmap#792] [Q&A]
Hello, Exist a place where try current development version compiled?
I use 2.6.16 and I want try current 2.6.17 or 2.8 version without try to recompile.

Thanks for your job

**Comment by albar965** [comment]
This is what  #697 is after but nobody has volunteered yet. :frowning: 

I plan to upload development builds from time to time at least to OneDrive and DropBox. These two can be automated halfway. I'll post in the Avsim forum and here once I have something.

About version numbering:
- All public releases (betas, rc and stable) are even numbers like 2.6 and future 2.8.
- Versions in development get odd numbers. Therefore, 2.7 is the current development branch.
- No plans for a 2.6.17 yet. This would require a serious bug fix of other severe issues.

Alex

**Reply by gabfus** [reply]
Thanks for info.
I wait news in avsim.

Great job
